src/components/Main.vue

```diff
@@ -99,25 +99,25 @@ export default defineComponent({
             </span>
           </div>
 
-          <div class="mb-4">
-            <div class="flex justify-between ound items-center bg-neutral-greyish-white border-0 rounded-xl
-               py-[22px] px-[24px]">
-              <button @click="decrementQuantity" class="text-primary-orange px-3">
+          <div class="flex flex-col md:flex-row gap-4">
+            <div class="flex justify-between items-center bg-neutral-greyish-white border-0 rounded-xl
+                py-[22px] px-[24px] md:p-0">
+              <button @click="decrementQuantity" class="text-primary-orange px-3 py-4">
```

Comment:
Icons and text in the quantity and add to cart buttons are very small, for user - it could be hard to press. 
From your code:

![Screenshot from 2023-04-24 10-34-56](https://user-images.githubusercontent.com/36459922/233929674-ca60f1d5-f5b4-4009-a6e5-7290ecdda4fe.png)

By the design:

![Screenshot from 2023-04-24 10-34-46](https://user-images.githubusercontent.com/36459922/233929739-209b4577-340c-4ad4-bfcf-4a4a5be20e24.png)